lua/telescope/actions/init.lua

```diff
@@ -413,7 +413,7 @@ end
 
 actions.smart_send_to_qflist = function(prompt_bufnr)
   local picker = action_state.get_current_picker(prompt_bufnr)
-  if table.getn(picker:get_multi_selection()) > 0 then
+  if #(picker:get_multi_selection()) > 0 then
```

Comment:
I don't like this. For me `#` is for string length and `table.getn` for table length. Makes the code cleaner in my opinion.

I know its deprecated and removed in lua 5.2 but that won't happen to luajit